controllers/properties-routes.js

```diff
@@ -0,0 +1,38 @@
+// Getting individual properties
```

Comment:
@LennonSimpson why is this file needed?  Why wouldn't we use the get one function in `controllers/api/property-routes`?

Reply:
I assumed this was for the front end, re: pages that show individual property details.